src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arraySorceString = sourceString
+    .split(';')
+    .map((string) => string.trim().split(':'));
+
+  const filteredSorceString = arraySorceString
+    .filter((innerArr) => innerArr.length === 2)
+    .map((element) => element.map((item) => item.trim()));
```

Comment:
The variable 'filteredSorceString' could be named more appropriately to reflect the filtered and trimmed style declarations, e.g., 'trimmedStyleDeclarations'.